hikari/impl/bot.py

```diff
@@ -396,7 +385,7 @@ async def handle(name: str, awaitable: typing.Awaitable[typing.Any]) -> None:
                     }
                 )
 
-        await self.dispatch(self.event_factory.deserialize_stopping_event())
+        await self._event_manager.dispatch(self._event_factory.deserialize_stopping_event())
```

Comment:
Isn't this inconsistent compared with how artificial events are dispatched in bot.py

Reply:
(also the event manager doesn't have managers for any artificial events anymore)